client/components/mma/cancel/cancellationSaves/digipack/ThankYouOffer.stories.tsx

```diff
@@ -0,0 +1,54 @@
+import type { Meta, StoryObj } from '@storybook/react';
```

Comment:
We could have a different storybook file per screen or rename this file to something more generic which has all of the scenarios in it.

Reply:
yeah I wasn't sure...I think since this had two scenarios I went with the former but we can always shove them into the same file later

Reply:
I can do as part of my next PR